packages/terra-functional-testing/CHANGELOG.md

```diff
@@ -2,11 +2,14 @@
 
 ## Unreleased
 
+* Breaking
+  * Upgraded to wdio v7
+  * Increased large viewport size to `1020`
+
 ## 1.9.0 - (June 18, 2021)
 
 * Fixed
   * Updated screenshot cleaning to match nested snapshot directories
-  * Updated to size the test page to the viewport size instead of the browser's window size to correctly match the corresponding breakpoint. This change may affect the size of existing screenshots, particularly in IE and firefox. The affected screenshots may need to be regenerated.
```

Comment:
I think we should keep the changelog instead of removing it since we did make this change on version 1.9.0 and it would be helpful to let consumers know what 1.9.0 would cause screenshot mismatches.